src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs

```diff
@@ -67,6 +68,23 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_FileSystemTypes(CustomC
                     .EndRowDefinition()
                 .EndTable());
 
+            yield return new FormatViewDefinition("childrenWithHardlink",
+                TableControl.Create()
+                    .GroupByProperty("PSParentPath", customControl: sharedControls[0])
+                    .AddHeader(Alignment.Left, label: "Mode", width: 7)
+                    .AddHeader(Alignment.Right, label: "LastWriteTime", width: 25)
+                    .AddHeader(Alignment.Right, label: "Length", width: 14)
+                    .AddHeader()
+                    .StartRowDefinition(wrap: true)
+                        .AddPropertyColumn("Mode")
+                        .AddScriptBlockColumn(@"
+                                        [String]::Format(""{0,10} {1,8}"", $_.LastWriteTime.ToString(""d""), $_.LastWriteTime.ToString(""t""))
```

Comment:
Seems like this is indented more than necessary

Reply:
I think I was just following the formatting/style of the rest of the file. Not sure what we want here.

Reply:
Oh, you are right! fixing.